README.md

```diff
@@ -39,6 +39,7 @@ The following environment variables are used to configure the script:
   for your team.
 * `ARCHIVEBOT_INACTIVITY_DAYS`: the number of days' inactivity after which to
   archive a channel (default: `30`).
+* `ARCHIVEBOT_NOTIFY`: a Slack user or channel (e.g. `#general` or `@tblair`) to notify when something goes wrong.
```

Comment:
Please wrap this line at 78 characters (or after the closing bracket, whatever that point it).